includes/wikia/api/NavigationApiController.class.php

```diff
@@ -32,30 +31,36 @@ public function getData() {
 		);
 	}
 
-	private function getChildren( $list, $i = 0 ){
-		$children = array();
-		$next = array();
+	private function getChildren( $list, $i = 0 ) {
+		$children = [ ];
+		$next = [ ];
 
-		$element = $list[$i];
+		if ( isset ($list[$i]) ) {
+			$element = $list[$i];
+		} else {
+			return [ ];
+		}
 
-		if ( isset( $element['children'] ) ) {
-			foreach( $element['children'] as $child ){
+		if ( isset($element['children']) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "children" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)